vi/news/_posts/2016-12-25-ruby-2-4-0-released.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: news_post
+title: "Ruby 2.4.0 Released"
+author: "naruse"
+translator:
+date: 2016-12-25 00:00:00 +0000
+lang: vi
+---
+
+Chúng tôi rất hân hạnh được thông báo về việc phát hành Ruby phiên bản 2.4.0.
+
+Ruby 2.4.0 là phiên bản stable đầu tiên của Ruby 2.4.0 gồm nhiều tín năng mới như:
+
+## [Giới thiệu cải tiến hash table (bởi Vladimir Makarov)](https://bugs.ruby-lang.org/issues/12142)
+
+Cải thiện cấu trúc bên trong của hash table (st_table)
+Improve the internal structure of hash table (st_table) bằng việc giới thiệu
```

Comment:
Is this `Improve the internal structure of hash table (st_table)` an undeleted phrase?